tests/StreamManager.test.js

```diff
@@ -0,0 +1,17 @@
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+
+describe("StreamManager", function () {
+  before(async () => {
+    const [admin, payer, payee1, payee2] = await ethers.getSigners()
+    this.admin = admin
+    this.payer = payer
+    this.payee1 = payee1
+    this.payee2 = payee2
+    this.token = '0xA02f6adc7926efeBBd59Fd43A84f4E0c0c91e832' // USDT on mumbai testnet
```

Comment:
@accompany1205 not sure how we are going to get test tokens for ourselves too.

Reply:
It's just a setup for future use. We can use our own if we have.

Reply:
@markholdex this is out of the scope. This is just to set up the testing framework